cypress.json

```diff
@@ -0,0 +1,4 @@
+{
+  "baseUrl" : "http://localhost:3000/",
```

Comment:
so in here you don't need to add `/` at the end b/c the common practice is to use target path with `/`

your:
```
"baseUrl" : "http://localhost:3000/",
cy.visit("shop")
```
common:
```
"baseUrl" : "http://localhost:3000",
cy.visit("/shop")
```

and it exactly shows that you are going to visit some path

Reply:
done